src/commands/size/index.ts

```diff
@@ -1,56 +1,79 @@
-import { ChatInputCommandInteraction, SlashCommandBuilder } from 'discord.js';
+import type { ChatInputCommandInteraction } from 'discord.js';
 import MersenneTwister from 'mersenne-twister';
-import { Command } from '../../utilities/command';
+
+import { Command } from '../../types/command.js';
+import type {
+  CommandBuilder,
+  CommandBuilderOutput
+} from '../../types/command-builder.js';
+
+enum DongSize {
+  Micro = 'micro',
+  Normal = 'normal',
+  Magnum = 'magnum'
+}
+
+/** Map of sixe to percentage chance for  */
+const DONG_SIZE_MAP: ReadonlyMap<DongSize, number> = new Map([
+  [DongSize.Micro, 10],
+  [DongSize.Normal, 75],
+  [DongSize.Magnum, 15]
+]);
+const SUM_OF_PROBABILITY = Array.from(DONG_SIZE_MAP)
+  .map(([, probability]) => probability)
+  .reduce((total, probability) => total + probability, 0);
+
+const SIZE_DECIMAL_PRECISION = 2;
 
 // TODO: Evaluate the spread of the modifier generation
 // TODO: Improve message formatting
 // TODO: Implement dong leaderboard
 // TODO: Allow people to regenerate their dong on occasion (need to redesign how hash works)
 // TODO: Add gifs maybe?
 
-export default class SizeCommand extends Command {
-  async build() {
-    return new SlashCommandBuilder()
-      .setName('size')
-      .setDescription('Measure the size of your dong!')
-      .addUserOption((option) =>
-        option.setName('user').setDescription('User to get size of')
-      );
+export class SizeCommand extends Command {
```

Comment:
TODO: This whole command doesn't work anymore and its cause I'm bad at programming

Reply:
I just reverted everything that was unnecessary. I'll refactor this later. Or not. Who knows.